mpas_analysis/ocean/nino34_timeseries.py

```diff
@@ -0,0 +1,198 @@
+from ..shared.plot.plotting import nino34_timeseries_plot, nino34_spectra_plot
+import pandas as pd
+import numpy as np
+from scipy import signal, stats
+
+from ..shared.io import NameList, StreamsFile
+from ..shared.io.utility import buildConfigFullPath
+from ..shared.climatology import climatology
+
+from ..shared.generalized_reader.generalized_reader \
+    import open_multifile_dataset
+
+from ..shared.timekeeping.utility import get_simulation_start_time, \
+    date_to_days, days_to_datetime
+
+
+def nino34_timeseries(config, streamMap=None, variableMap=None, SSTregions=None):
+    """
+    Performs analysis of the time-series output of sea-surface temperature
+    (SST).
+
+    config is an instance of MpasAnalysisConfigParser containing configuration
+    options.
+
+    If present, streamMap is a dictionary of MPAS-O stream names that map to
+    their mpas_analysis counterparts.
+
+    If present, variableMap is a dictionary of MPAS-O variable names that map
+    to their mpas_analysis counterparts.
```

Comment:
would you mind adapting this doc string to the standard format we have been trying to adhere to in the last few PRs? You can use the MOC script as an example. thanks.